config/fcos/v1_6_exp/validate.go

```diff
@@ -15,13 +15,26 @@
 package v1_6_exp
 
 import (
+	"regexp"
+
 	"github.com/coreos/butane/config/common"
 	"github.com/coreos/ignition/v2/config/util"
 
 	"github.com/coreos/vcontext/path"
 	"github.com/coreos/vcontext/report"
 )
 
+var regex = regexp.MustCompile(`^/(etc|var)/.*`)
+
+func (conf Config) Validate(c path.ContextPath) (r report.Report) {
+	for _, fs := range conf.Storage.Filesystems {
+		if (fs.Path == nil) || (!regex.MatchString(*fs.Path)) {
```

Comment:
Hmm, I am not a big fan of '!' logic, (total nit) but could we maybe try something like if it is good then we continue, otherwise error?

Reply:
what do you think of this?
```
func (conf Config) Validate(c path.ContextPath) (r report.Report) {
	for _, fs := range conf.Storage.Filesystems {
		if util.NilOrEmpty(fs.Path) {
			r.AddOnError(c.Append("storage"), common.ErrMountPointForbidden)
		} else if (regex.MatchString(*fs.Path)){
			continue
		}else{
			r.AddOnError(c.Append("storage"), common.ErrMountPointForbidden)
		}
	}
	return
}
```

Reply:
Eh nvm disregard.